### DISCUSSION TITLE
Some repo not able to create the discussion

### DISCUSSION THREAD

**Discussion by ericlou168** [community/community#3545] [Discussions]
Some repo after setup the discussion and the create button not exist.

**Comment by ThomasAunvik** [comment]
Same issue, ive setup Discussions in my repo, but is unable to create one, i then get this error:
![image](https://user-images.githubusercontent.com/9968151/111126967-9441b700-8573-11eb-83d7-fd431178fb47.png)
by pressing this button "Set up discussions".
![image](https://user-images.githubusercontent.com/9968151/111127089-b76c6680-8573-11eb-8c74-d0b5f6e5b93a.png)

Though my co-worker is still able to create discussions, but i can't?

**Comment by PranamBhat** [comment]
Hi @ericlou168 

Under settings, make sure you have enabled it to "Public"

**Reply by ericlou168** [reply]
You mean change repo to public access?

**Reply by PranamBhat** [reply]
Yes!

**Reply by ericlou168** [reply]
The thing is some of my repos are private but still able to have discussion.

**Comment by PranamBhat** [comment]
This is the scenario. In the settings, make sure you have enabled these  : 


![image](https://user-images.githubusercontent.com/32120033/111253308-f9abab80-8638-11eb-8681-1f19f6125d5b.png)


In the "Manage Access", check these settings:

![image](https://user-images.githubusercontent.com/32120033/111253650-8c4c4a80-8639-11eb-9723-bf81b21a559b.png)


Also, in the "Moderation Settings" and "Interaction Limits", check these settings:

![image](https://user-images.githubusercontent.com/32120033/111253447-3677a280-8639-11eb-928e-6259c56850cb.png)
